src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -88,4 +93,68 @@ public function provideDefaultRoles(DefaultRoleEventInterface $event) {
     $event->addRole($role);
   }
 
+  /**
+   * Acting upon group creation and attach an og audience field to the user.
```

Comment:
maybe something like: `Upon group creation, add an OG audience field to the user if it doesn't exist.`